wg-manifests/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Manifests Charter
+
+This charter adheres to the conventions, roles and organization management
+outlined in [wg-governance].
+
+## Scope
+
+- Provide a catalog (centralized repository) of Kubeflow application manifests.
```

Comment:
We won't maintain the current manifests. We will structure new manifests based on the principles laid in this PR. The same manifests repo will be used.

Reply:
@Jeffwan if you are asking this due to kfctl, I think there are two options we can explore together:
1. You either keep/copy whatever current manifest you need that is kfctl-specific in the kfctl repo
2. You create a new overlay in the kfctl repo, which is kfctl-specific on top of the new manifests of this repo. This will make things much cleaner and maintainable even for kfctl.

Reply:
> We won't maintain the current manifests. We will structure new manifests based on the principles laid in this PR. The same manifests repo will be used.

@yanniszark who is maintaining the manifests you or the application owners? My expectation is that this repo is just a mirror of whatever is upstream e.g.

/pipelines = https://github.com/kubeflow/pipelines/tree/master/manifests
/kfserving = https://github.com/kubeflow/kfserving/tree/master/config
/katib = https://github.com/kubeflow/katib

It looks like for the other applications they may not have upstream manifests yet.

So my expectation is that this WG is

1. Building automation to copy over the manifests
1. Coordinating with app WGs to create their upstream manifests

Does that match your understanding?

Reply:
Yes it does. I will soon update the charter to reflect that.